### ISSUE TITLE
Day 13 - can't tweet as the rsps says limited access to the api

### ISSUE THREAD

**Issue by ashutoshsao** [MichaelCade/90DaysOfDevOps#450]
this is the message i received on the terminal, maybe the methods have changed to post using the twitter api

twitter: 453 You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/portal/product

**Comment by MichaelCade** [comment]
Yes the Twitter API has now changed since it went to X. I should look at the example use case during the golang phase.

**Comment by MichaelCade** [comment]
I have created an issue to try and resolve and update this. https://github.com/MichaelCade/90DaysOfDevOps/issues/476
